www/js/services.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+/* Notes on forecast.io's API:
+ *  - daily.data[0] is today
+ */
+
+weatherApp.constant('FORECASTIO_KEY', '74f28a5eb37ac7f45ca31633870c57b0');
+
+/* weatherStore holds data for weatherData service.
+ */
+weatherApp.factory('weatherStore', function() {
+    var weatherStore = {
+	current : {}
+    };
+
+    return weatherStore;
+});
+
+/* weatherData service gets weather data (current, forecasts, and
+ * historical), processes it as necessary, and provides it to controllers.
+ */
+weatherApp.service('weatherData', ['$q', '$resource', '$http',
+				   'FORECASTIO_KEY', 'weatherStore', '$filter',
+    function($q, $resource, $http, FORECASTIO_KEY, weatherStore, $filter) {
+	this.getCurrentWeather = function(lat, lng) {
+	    var url = 'https://api.forecast.io/forecast/'
+		+ FORECASTIO_KEY + "/" + lat + ',' + lng;
+
+	    // JSONP is only needed for "ionic serve".
+	    // Simpler $http.get(url) works on devices.
+	    return $http.jsonp(url + '?callback=JSON_CALLBACK').then(
+		function success(resp) {
+		    weatherStore.current = resp.data;
+		    console.log('GOT CURRENT');
+		    console.dir(weatherStore.current);
+		},
+		function failure(error) {
+		    alert('Unable to get current conditions');
+		    console.error(error);
+		});
+	};
+
+  this.getLocation = function() {
```

Comment:
This is code that I got from professor Tuck's example code, so I am not 100% sure what this does. So I just left it as is.